app/controllers/articles_controller.rb

```diff
@@ -27,7 +27,9 @@ def create
         end    
         #render plain: params[:article].inspect
     end
-
+    def userless_controller
+        puts 'nothing'
```

Comment:
Use 2 (not 4) spaces for indentation.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not write to stdout. Use Rails' logger if you want to log.